CONTRIBUTING.md

```
@@ -1,80 +1,245 @@
 
-```shell
-export BUILD_LOCALLY=1
-make
+All commits must be signed off (DCO):
+
+```bash
+git commit -s -m "Your commit message"
+```
+
+To amend:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)